xc/xc7/tests/install_test/CMakeLists.txt

```diff
@@ -5,8 +5,6 @@ add_test(NAME binary_toolchain_test
 	PATH=${INSTALLATION_DIR_BIN}:$ENV{PATH}
 	${CMAKE_COMMAND} -E env
 	PYTHONPATH=${PRJXRAY_DIR}:${PRJXRAY_DIR}/third_party/fasm
-	XRAY_FASM2FRAMES=${PRJXRAY_DIR}/utils/fasm2frames.py
-	XRAY_TOOLS_DIR=${symbiflow-arch-defs_SOURCE_DIR}/build/third_party/prjxray/tools
-	XRAY_DATABASE_DIR=${PRJXRAY_DB_DIR}
+	FRAMES2BIT=$<TARGET_FILE:xc7frames2bit>
```

Comment:
Should probably still be setting `DATABASE_DIR` given that `PRJXRAY_DB_DIR` exists?

Reply:
Given that this tests the toolchain it would probably be better to leave the DATABASE_DIR unset, so that we are sure that the prjxray-db package is working properly

Reply:
I understand what you are saying, but we have an inconsistency then.  We definitely should not have two copies of the prjxray-db during the CI, and I think it is more important to respect the `PRJXRAY_DB_DIR` CMake cache variable.  Still thinking.

Reply:
I believe we should respect `PRJXRAY_DB_DIR ` in this case.  The cache variable can default to the result of `prjxray-config`, but if the use specifies another path, it should be used here.

You also need to remove the prjxray-db submodule.

Reply:
All right, I can than remove the dependency and override the DATABASE_DIR. Than we'll update that in a new PR in the future